nilearn/plotting/html_surface.py

```diff
@@ -21,7 +21,8 @@ class SurfaceView(HTMLDocument):
 
 
 def _get_vertexcolor(surf_map, cmap, norm,
-                     absolute_threshold=None, bg_map=None):
+                     absolute_threshold=None, bg_map=None,
+                     bg_on_data=False, scale_bg_map=True, darkness=1):
```

Comment:
is it good to specify defaults in low-level functions ? Maybe it would be better to only specify them in higher-level functions.